### PR TITLE
Update eep-0048.md

### PR DESCRIPTION
**Pull request by josevalim**
* Rename Language to BeamLanguage to avoid confusion
* Store translations in a map keyed by language

### CODE REVIEW

File: eeps/eep-0048.md [hunk 1]
```diff
@@ -101,32 +101,34 @@ the type specification below:
 
     {docs_v1,
      Anno :: erl_anno:anno(),
-     Language :: atom(),
+     BeamLanguage :: atom(),
      Format :: mime_type(),
-     ModuleDoc :: binary() | none | hidden,
+     ModuleDoc :: %{DocLanguage => DocString} | none | hidden,
```
Comment (RaimoNiskanen) [inline eeps/eep-0048.md]: Is this Erlang term format or Elixir?  If Erlang should this not be `#{DocLanguage => DocString}`?

File: eeps/eep-0048.md [hunk 2]
```diff
@@ -152,9 +154,10 @@ For each entry in `Docs`, we have:
     may be rendered as as a single line or two lines. It exists
     exclusively for exhibition purposes
 
-  * `Doc` - a binary with the documentation or the atom `none`
-    in case there is no documentation or the atom `hidden` if
-    documentation has been explicitly disabled for this entry
+  * `Doc` - a map with the documentation language as key, such as
+    "en" or "pt_BR", and the documentation as a binary value.  It may be
```
Comment (RaimoNiskanen) [inline eeps/eep-0048.md]: Is the key a string or a binary.  If binary and if this is Erlang Term format the examples should be `<<"en">>` and `<<"pt_BR">>`, right?

File: eeps/eep-0048.md (hunk 1 above)
Comment (josevalim) [reply eeps/eep-0048.md]: Good catch. It is Elixir's. 😰

**Comment by josevalim** [comment]
Thanks @RaimoNiskanen. Both of your comments have been addressed and I force pushed.

File: eeps/eep-0048.md [hunk 3]
```diff
@@ -101,32 +101,35 @@ the type specification below:
 
     {docs_v1,
      Anno :: erl_anno:anno(),
-     Language :: atom(),
+     BeamLanguage :: atom(),
      Format :: mime_type(),
-     ModuleDoc :: binary() | none | hidden,
+     ModuleDoc :: %{DocLanguage := DocString} | none | hidden,
```
Comment (essen) [inline eeps/eep-0048.md]: Isn't `%{...}` the Elixir notation of maps, in which case it should be replaced with `#{...}`?

**Comment by RaimoNiskanen** [comment]
I still see the map typespec for `ModuleDoc` and `Doc` use a `%` sign instead of a `#`

File: eeps/eep-0048.md [hunk 4]
```diff
@@ -101,32 +101,35 @@ the type specification below:
...
          Metadata :: map()
-        }]}
+        }]} when DocLanguage :: binary(),
+                 DocString :: binary()
 
 where in the root tuple we have:
 
   * `Anno` - annotation (line, column, file) of the module documentation
-    or of the definition itself (see erl_anno)
+    or of the definition itself (see `erl_anno`)
```
Comment (RaimoNiskanen) [inline eeps/eep-0048.md]: You have somehow managed to get two non-breaking-space in the beginning of this line: [32,160,32,160]

File: eeps/eep-0048.md [hunk 5]
```diff
@@ -101,32 +101,35 @@ the type specification below:
...
     `erlang`, `elixir`, `lfe`, `alpaca`, etc
 
   * `Format` - the mime type of the documentation, such as "text/markdown"
     (see the FAQ for a discussion on this field)
 
-  * `ModuleDoc` - a binary with the documentation or the atom `none`
-    in case there is no documentation or the atom `hidden` if
-    documentation has been explicitly disabled for this entry
+  * `ModuleDoc` - a map with the documentation language as key, such as
+    `<<"en">>` or `<<"pt_BR">>`, and the documentation as a binary value.
```
Comment (RaimoNiskanen) [inline eeps/eep-0048.md]: You have somehow managed to get two non-breaking-space in the beginning of this line: [32,160,32,160]

File: eeps/eep-0048.md [hunk 6]
```diff
@@ -152,9 +155,11 @@ For each entry in `Docs`, we have:
     may be rendered as as a single line or two lines. It exists
     exclusively for exhibition purposes
 
-  * `Doc` - a binary with the documentation or the atom `none`
-    in case there is no documentation or the atom `hidden` if
-    documentation has been explicitly disabled for this entry
+  * `Doc` - a map with the documentation language as key, such as
+    `<<"en">>` or `<<"pt_BR">>`, and the documentation as a binary value.
```
Comment (RaimoNiskanen) [inline eeps/eep-0048.md]: You have somehow managed to get two non-breaking-space in the beginning of this line: [32,160,32,160]

File: eeps/eep-0048.md (hunk 3 above)
Comment (josevalim) [reply eeps/eep-0048.md]: Haha, yes... too much context switching.

**Comment by josevalim** [comment]
Apologies... force pushed once more.

@RaimoNiskanen I have ported the EEP from a Google Docs document... so I am guessing that was the cause for the non-breaking white spaces. I have ran a script to remove them all.

**Pull request merged**